core/modules/SConscript

```diff
@@ -95,6 +95,14 @@ def standardModule(env, exclude=None, module=None, test_libs=None, unit_tests=No
                 then use this parameter to pass list (or space-separated
                 string) of application names. Can be empty string or list to
                 not run any unit tests.
+    @param bin_cc_targets: a dictionary defining whgich binary products (not
+                the unit tests should be built out of C/C++ files. Each key
```

Comment:
Opening parenthesis on previous line is not closed, I guess it should go after "tests".

Reply:
ok